### DISCUSSION TITLE
What is the best OS (Operating System) for you?

### DISCUSSION THREAD

**Discussion by rfLENtlr** [NAIST-SE/AttractingContributors#911] [2023 Attraction Contributions]
Could you add the best OS for you to my list?
If you make a pull request, I'll do a pull request too. My list is below.
https://github.com/rfLENtlr/AttractingContributors-OS

あなたにとって最高のOSを私のリストに追加してほしいです。
プルリクエストを送ってくれたら、私もプルリクエストを送ります。
https://github.com/rfLENtlr/AttractingContributors-OS

**Comment by bconfessor** [comment]
I prefer Windows, I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md
